pkg/rabbitqq/client.go

```diff
@@ -17,7 +17,10 @@ type Client interface {
 	Add(key string, value string) (bool, error)
 	Remove(key string) (bool, error)
 	Get(key string) (*string, error)
-	GetAll() (map[string]string, error)
+	GetAll() ([]struct {
```

Comment:
Declare named struct. You reuse it multiple times. Or just use map[string]string as before